cmd/terraform_native_commands.go

```diff
@@ -0,0 +1,201 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// getTerraformCommands returns an array of statically defined Terraform commands with flags
+func getTerraformCommands() []*cobra.Command {
+	// List of Terraform commands
+	return []*cobra.Command{
+		{
+			Use:   "plan",
+			Short: "Show changes required by the current configuration",
+			Long:  "Generate an execution plan, which shows what actions Terraform will take to reach the desired state of the configuration.",
```

Comment:
I want to add an `Example` section here showing how to use plan, since it's one of the most common use-cases.

Reply:
done